frontend/pages/[resourceType].tsx

```diff
@@ -104,8 +102,24 @@ export const getServerSideProps: GetServerSideProps<{
     // Measure Repository should not provide a bundle without an entry
     throw new Error('Measure Repository bundle has no entry.');
   }
-  const ids = bundle.entry.map(entry => entry.resource?.id);
+  const resources = bundle.entry as fhir4.BundleEntry<FhirArtifact>[];
```

Comment:
We can avoid double casting the bundle by casting it higher up to `fhir4.Bundle<FhirArtifact>`.